### ISSUE TITLE
Exception Thrown when trying to use NLP Simple CoreNLP

### ISSUE THREAD

**Issue by Nodice** [sergey-tihon/Stanford.NLP.NET#44]
1.6.0 Simple reference here: http://stanfordnlp.github.io/CoreNLP/simple.html#simple-corenlp

> Exception java.lang.NoClassDefFoundError "com.google.protobuf.GeneratedMessage$ExtendableMessage"

Exception Thrown at: `Document doc = new Document(inText)`

``` csharp
using edu.stanford.nlp.simple;
public List<string> SentenceDetect(string inText)
{
    Document doc = new Document(inText);
    var sents = doc.sentences();
    var l = ((Sentence)sents.get(0)).lemmas();
    return new List<string>(sentenceParser.SentenceDetect(inText));
}
```

**Comment by nerzid** [comment]
I've the same issue.

**Comment by nerzid** [comment]
I've just solved this issue via adding the following dependency to pom.xml
```
<!-- https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java -->
<dependency>
    <groupId>com.google.protobuf</groupId>
    <artifactId>protobuf-java</artifactId>
    <version>3.0.2</version>
</dependency>
```

**Comment by sergey-tihon** [comment]
Wow... CoreNLP uses `pom.xml` to load dependencies ... Very strange!
Are you sure that you use `.NET` version of `CoreNLP` ?)
In which directory you created `pom.xml` file? (I do not ship it with NuGet package)

**Comment by sartan** [comment]
FYI- supposedly will be fixed in CoreNLP 3.7.0

https://github.com/stanfordnlp/CoreNLP/issues/171#issuecomment-254599194

**Comment by sergey-tihon** [comment]
@sartan you may verify it https://www.nuget.org/packages/Stanford.NLP.CoreNLP/3.7.0-beta

**Comment by jsnreddy** [comment]
I have tried the StanfordCoreNLP 3.7.0 but I am still facing this problem. I have tried adding protobuf-java-3.0.2.jar to the classpath but still there is an error when trying to load coreNLP server.

Also tried 3.6.0 with the protobuf-java-3.0.2.jar in the classpath, I was facing the same problem there as well.

Is this fixed in the latest jar available for download. Or should I compile from the latest sources?
Any suggestions?
